sdk/extensions/chronopost/karrio/mappers/chronopost/mapper.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Tuple
+from karrio.core.utils.serializable import Serializable, Deserializable
+from karrio.api.mapper import Mapper as BaseMapper
+from karrio.core.models import (
+    RateDetails,
+    RateRequest,
+    TrackingRequest,
+    TrackingDetails,
+    Message,
+)
+from karrio.universal.providers.rating import parse_rate_response, rate_request
```

Comment:
```suggestion
```

We will need to implement the chronopost-specific rate request and response so we can remove this import.